ts/packages/cli/src/commands/export.ts

```diff
@@ -156,6 +186,11 @@ export default class Export extends BaseCommand<typeof Export> {
     this.printStdout(formattedConfigs);
   }
 
+  applyCasing(result: { [key: string]: string }) {
```

Comment:
this should be with the same keyMutations function of the --prefix PR

Reply:
Done. Thanks